src/app/zap-templates/partials/im_command_handler_cluster_commands.zapt

```diff
@@ -8,7 +8,15 @@ wasHandled = emberAf{{asUpperCamelCase parent.name}}Cluster{{asUpperCamelCase na
 {{#if (zcl_command_arguments_count this.id)}}
 expectArgumentCount = {{ zcl_command_arguments_count this.id }};
 {{#zcl_command_arguments}}
-{{asUnderlyingZclType type}} {{asSymbol label}};
+  {{#if isArray}}
+    {{asUnderlyingZclType type}} {{asSymbol label}};
+  {{else}}
```

Comment:
`{{else if...}}`?

Reply:
No, because you can't do `else if_is_struct type`, sadly.

Reply:
Ah, I missed the underscore.